docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -33,7 +33,50 @@ Keeping informed
 To track changes that affect collections:
 
 * Join the `Collection Maintainers & Contributors forum group <https://forum.ansible.com/g/CollectionMaintainer>`_.
-* Subscribe to the `Bullhorn <https://forum.ansible.com/c/news/bullhorn/17>`_ Ansible contributor newsletter.
+* Subscribe to the :ref:`bullhorn` Ansible contributor newsletter.
+
+.. _coll_wg_reqs:
+
+Communication and Working Groups
+================================
+
+Forum overview
+--------------
+
+The :ref:`ansible_forum` is our default communication platform.
```

Comment:
Have we officially decided that? Maybe

```suggestion
The :ref:`ansible_forum` is our asynchronous default communication platform.
```

is more accurate.

Reply:
I think it's a default platform in any sense, @gundalow is that correct?

Reply:
Hmm, isn't this somewhat related to the proposal https://forum.ansible.com/t/6812 ? I'm really not sure what to write here, since so far we haven't adopted that proposal (and it's still in discussion).

Reply:
> Hmm, isn't this somewhat related to the proposal https://forum.ansible.com/t/6812 ? I'm really not sure what to write here, since so far we haven't adopted that proposal (and it's still in discussion).

Right. I think my suggestion reflects the status quo -- we have been slowly migrating from the mailing lists and Github Issues/Discussions to the forum for *asynchronous* discussion -- but even that hasn't been made official.

Reply:
Committed, thanks